tests/websocket-python-clients/simple_websocket.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+import websockets
+
+async def connect():
+    uri = "ws://localhost:8080"
+    async with websockets.connect(uri) as websocket:
+        # Send a message to the server
+        await websocket.send("Hello, Server!")
+        print("Sent: Hello, Server!")
+
+        # Receive a message from the server
+        response = await websocket.recv()
+        print(f"Received: {response}")
+
+asyncio.get_event_loop().run_until_complete(connect())
```

Comment:
if we can't do this with `curl -F file=@/path/to/your/file.txt http://localhost:8080/upload ` something is wrong.

Reply:
As discussed, received message:

```
curl -F file=@/home/jpn323/workspace/ut-core/framework/ut-control/tests/websocket-clients/example.yaml ws://localhost:8080
curl: (1) Protocol "ws" not supported or disabled in libcurl

```

Reply:
https://github.com/rdkcentral/ut-control/issues/3